### PR TITLE
Migrate deprecated http function to client (api call)

### LINKED ISSUE #4524: Integrate api call with client.js, http.js handler is deprecated
### Description
Lisk wallet now support multiple application, we introduce Client Singleton class to handle api call in rest or rpc depend of the config, we used it most the time in react query, but some of the utils calling the api directly with http.js  they need to integrate with new client

### Acceptance Criteria
- Remove the http.js file
- Update utiles with client
- Adopt Unit Tests

### PR DESCRIPTION
**Pull request by soroushm**
### What was the problem?

This PR resolves #4524 

### How was it solved?
its in progress

### How was it tested?
Unit test

### CODE REVIEW

File: src/modules/blockchainApplication/manage/components/AddApplicationList/index.js [hunk 1]
```diff
@@ -1,39 +1,10 @@
...
-      data: mockBlockchainApplications,
-    }),
-    getApiParams: (state) => ({
-      network: state.network,
-    }),
-    transformResponse: response => response.data,
-    autoload: false,
-  },
-};
-
```
Comment (ikem-legend) [inline src/modules/blockchainApplication/manage/components/AddApplicationList/index.js:35]: @soroushm The API util here was removed but the child component is still expecting the props removed. The child component can either be completely refactored too or a TODO should be added here for that

File: src/modules/transaction/api/index.test.js [hunk 2]
```diff
@@ -11,10 +11,8 @@ import accounts from '@tests/constants/wallets';
 import { genKey, blsKey, pop } from '@tests/constants/keys';
 import { mockAppTokens } from '@tests/fixtures/token';
 import {
-  getTransaction,
```
Comment (ikem-legend) [inline src/modules/transaction/api/index.test.js:14]: @soroushm `getTransaction` was removed from the test but the util in the index file wasn't removed

File: src/modules/transaction/manager/blockDetailsTransactionManger.js [hunk 3]
```diff
@@ -1,21 +1,8 @@
 /* istanbul ignore file */
 import { compose } from 'redux';
 import { withTranslation } from 'react-i18next';
-import withData from 'src/utils/withData';
 import BlockDetailsTransactions from '../components/BlockDetailsTransactions/BlockDetailsTransactions';
-import { getTransactions } from '../api';
 
 export default compose(
```
Comment (ikem-legend) [inline src/modules/transaction/manager/blockDetailsTransactionManger.js]: @soroushm If `withTranslation` is the only HOC used, then there's no need for `compose`

**Review by ikem-legend — COMMENTED** [review]
Just a few minor fixes and it's good to go

File: src/modules/transaction/api/index.test.js (hunk 2 above)
Comment (soroushm) [reply src/modules/transaction/api/index.test.js:14]: Done

File: src/modules/blockchainApplication/manage/components/AddApplicationList/index.js (hunk 1 above)
Comment (soroushm) [reply src/modules/blockchainApplication/manage/components/AddApplicationList/index.js:35]: oh, missed that its need to use react-query

File: src/modules/transaction/manager/blockDetailsTransactionManger.js (hunk 3 above)
Comment (eniolam1000752) [reply src/modules/transaction/manager/blockDetailsTransactionManger.js]: We don't even need translation here

File: src/modules/transaction/manager/recentTransactionManager.js [hunk 4]
```diff
@@ -1,22 +1,5 @@
...
-      return {
-        address,
-        network: state.network,
-      };
-    },
-    defaultData: [],
-    transformResponse: response => response.data.splice(0, 5),
-  },
-})(withTranslation()(RecentTransaction));
+export default withTranslation()(RecentTransaction);
```
Comment (eniolam1000752) [inline src/modules/transaction/manager/recentTransactionManager.js:5]: We don't need translation here we can use 
`useTranslation` in the RecentTransaction component

**Review by eniolam1000752 — CHANGES_REQUESTED** [review]

File: src/modules/transaction/manager/recentTransactionManager.js (hunk 4 above)
Comment (soroushm) [reply src/modules/transaction/manager/recentTransactionManager.js:5]: yeah, but its just out of scope of this pr, we have a different task to fix this transaction recent

**Review by eniolam1000752 — APPROVED** [review]

**Pull request merged**